### DISCUSSION TITLE
Can't change configuration to allow big file uploads. 413 error

### DISCUSSION THREAD

**Discussion by edzzn** [EasyEngine/easyengine#1630] [Support Requests]
Hello I want to upload some big files to WordPress and I can't quite find a correct solution: 

![image](https://user-images.githubusercontent.com/14936466/141147228-2970a0f6-f507-428e-96b1-d963f532c325.png)

I had made the following changes to a new site.

1) in `/opt/easyengine/sites/mysite.com/config/nginx/conf.d/main.conf` I changed the following:

```
...
# client_max_body_size 100m;
client_max_body_size 1000m;
...
```

2) in `/opt/easyengine/sites/mysite.com/config/php/php/conf.d/custom.ini` I had changed

```
...
upload_max_filesize = 1000M
post_max_size = 1000M
```
3) I ran:
`ee site restart mysite.com`

As far as I am aware this should fix the problem, but it persists. Thanks for a guide or response.

**Comment by cerbenus** [comment]
I would enable debug mode in wp-config.php:
```
define( 'WP_DEBUG', true );
// Enable Debug logging to the /wp-content/debug.log file
define( 'WP_DEBUG_LOG', true );
```
Then SFTP into this debug.log file. It will tell you what is going on behind the scenes.

**Reply by cerbenus** [reply]
Also, you shouldn't be modifying:

`/opt/easyengine/sites/mysite.com/config/nginx/conf.d/main.conf`

as it is clearly stated in the comment on LoC 1.
Use custom user.conf instead.

**Comment by tmsbrasil** [comment]
Hi you can change it on '/opt/easyengine/sites/<your-site>/config/php/php/conf.d' editing custom.ini, then after run 'ee site reload <your-site>'

**Comment by steve231293** [comment]
Hello everyone,
I have the same problem. For now it limited 100MB.
I want to upload file size about 300 MB

I edited value of **client_max_body_size** in 2 following files but without success:
- /opt/easyengine/sites/my-site.com/config/nginx/nginx.conf
- /opt/easyengine/sites/my-site.com/config/nginx/conf.d/main.conf

```
client_max_body_size 300m;
client_body_timeout 900s;
```

I also updated file:
/opt/easyengine/sites/my-site.com/config/php/php/conf.d/custom.ini

; Custom PHP settings

```
upload_max_filesize = 1024M
post_max_size = 1024M
memory_limit = 512M
max_input_time = -1
max_execution_time = 7200
max_input_vars = 15000
max_file_uploads = 500

[mail function]
sendmail_path = /usr/bin/msmtp -t
```

Please help me.
Thank you so much

**Comment by steve231293** [comment]
Hello @edzzn ,
Did you fix the problem yet?

**Comment by marcoandrei** [comment]
Hello @edzzn and @steve231293! Could you fix the problem?

**Comment by lmorit** [comment]
Hi, did @edzzn of @steve231293 found an answer. Does this has anything todo with the reverse proxy being used?
